spec.html

```diff
@@ -11832,14 +11832,29 @@ <h1>Execution Contexts</h1>
             Generator
           </td>
           <td>
-            The Generator that this execution context is evaluating.
+            The Generator that is associated with this execution context.
           </td>
         </tr>
       </table>
     </emu-table>
-    <p>In most situations only the running execution context (the top of the execution context stack) is directly manipulated by algorithms within this specification. Hence when the terms “LexicalEnvironment”, and “VariableEnvironment” are used without qualification they are in reference to those components of the running execution context.</p>
+    <p>To express the semantics of generators, this document uses steps of the form “<dfn id="transfer-control" variants="transfers control,transfer of control">Transfer control</dfn> from _aContext_ to _bContext_, passing _completion_”, where _aContext_ and _bContext_ are execution contexts and _completion_ is a Completion Record. This means:</p>
```

Comment:
```suggestion
    <p>To express the semantics of generators, this document uses steps of the form “<dfn id="transfer-control" variants="transfer control,transfers control,transfer of control">Transfer control</dfn> from _aContext_ to _bContext_, passing _completion_”, where _aContext_ and _bContext_ are execution contexts and _completion_ is a Completion Record. This means:</p>
```

For "If ..., transfer control ...".

Reply:
To be clear, no such step exists in this PR, you're just allowing for the possibility that it might in the future?

The construction `If ..., transfer control ...` seems unlikely to me, since (as proposed) every "Transfer control" step is immediately followed, at the same level of indentation, by a Note-step that talks about control being transferred back or not, and then usually by a Let-step that gets the result of the back-transfer. (Plus there's typically some execution context stuff at the same level.)